README.md

```diff
@@ -794,6 +794,24 @@
 
     ```
 
+  - Remove all trailing whitespace
+
+    ```javascript
+    // bad
+    function() {∙∙
+      var name;
+    ∙∙
+      // ... stuff ...
+    }
+
+    // good
+    function() {
+      var name;
+
+      // ... stuff ...
+    }
+    ```
+
```

Comment:
This is a good call. It would be the most excellent if we could also include guides on how to set up automatic trailing space removal in various IDEs/text editors.